pkg/controllers/deprovisioning/metrics.go

```diff
@@ -63,12 +63,12 @@ var (
 		},
 		[]string{actionLabel, deprovisionerLabel},
 	)
-	deprovisioningEligibleMachinesGauge = prometheus.NewGaugeVec(
+	deprovisioningEligibleNodesGauge = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: metrics.Namespace,
 			Subsystem: deprovisioningSubsystem,
-			Name:      "eligible_machines",
-			Help:      "Number of machines eligible for deprovisioning by Karpenter. Labeled by deprovisioner",
+			Name:      "eligible_nodes",
```

Comment:
should we be calling it `eligible_node_claims` here and likewise in other places?